src/artemis/UDC_move.to.default_state.py

```diff
@@ -0,0 +1,106 @@
+from bluesky import RunEngine
+from devices.I03Smargon import I03Smargon
+from devices.backlight import Backlight
+from devices.BeamStop import BeamStop
+from devices.miniap import MiniAperture
+from devices.FluorescenceDetector import FluorescenceDetector
+from devices.Scintillator import Scintillator
+from devices.Scatterguard import Scatterguard
+from devices.GonioLowerStages import GonioLowerStages
+from devices.Cryostream import Cryo
+from devices.CTAB import CTAB
+from bluesky.plan_stubs import mv, null
+import bluesky.plan_stubs as bps
+import bluesky.plans as bp
+from datetime import datetime
+
+# Moves beamline motors to known positions prior to UDC start. Logic order: Check cryo backpressure and temperature; Move scintillator stick out safely; Move fluorescence detector out and set CTAB to 0; Move Chi to 0; Move rest of Smargon and lower gonio stages to 0; Move beamstop to robot load position; Move miniap safely to robot load position; move cryostream in.
+
+sgon = I03Smargon(name="I03Smargon", prefix="BL03I")
+backlight = Backlight(name="Backlight", prefix="BL03I")
+mapt = MiniAperture(name="MiniAperture", prefix="BL03I")
+bs = BeamStop(name="BeamStop", prefix="BL03I")
+fluo = FluorescenceDetector(name="FluorescenceDetector", prefix="BL03I")
+scin = Scintillator(name="Scintillator", prefix="BL03I")
+sg = Scatterguard(name="Scatterguard", prefix="BL03I")
+gonp = GonioLowerStages(name="GonioLowerStages", prefix="BL03I")
+cryo = Cryo(name="Cryo", prefix="BL03I")
+ctab = CTAB(name="CTAB", prefix="BL03I")
+
+
+
+sgon.wait_for_connection()
+backlight.wait_for_connection()
+mapt.wait_for_connection()
+bs.wait_for_connection()
+fluo.wait_for_connection()
+scin.wait_for_connection()
+sg.wait_for_connection()
+gonp.wait_for_connection()
+cryo.wait_for_connection()
+ctab.wait_for_connection()
+
+
+IN=1
+OUT=0
+
+RE=RunEngine({})
+
+def make_scin_safe():
+
+	if (scin.y.position < 1 and scin.z.position < 1):
+		pass
+	else:
+		yield from bps.mv(scin.y, 0.1, mapt.x, -4.91, sg.x, -4.75) 
+		yield from bps.mv(scin.z, 0.1)
+
+def put_bs_in():
+	if (bs.z.position >25 and bs.y.position > 25):
+		pass
+	else:
+		yield from bps.mv(bs.z, 30)
+		yield from bps.mv(bs.y, 45, bs.x, 1.6) 
+		
+
+def check_temp():
+	cryo_temp = cryo.temp.get()
+	if cryo_temp > 110:
+		print ("temperature too high - ",cryo_temp)  # TODO abort here
+	else:
+		pass
+	yield from null()
```

Comment:
I suggested this line to make it work without much explanation so here is some. 

You can think of Bluesky plans as providing a list (actually a [generator](https://wiki.python.org/moin/Generators)) of small operations we're going to do e.g. a `rd` or a `mv`. It is useful to provide the list rather than doing the operations as then we can do things like ask a plan what operations it's going to perform without actually running them on real hardware, it also makes pausing and rewinding plans possible. When you're doing `yield from check_temp` in this context it's sort of like saying "Add the operations that check_temp returns to the list of operations to do". It was throwing an error because without the `yield from null()` here `check_temp` was returning no operations. `Null` is a sort of dummy operation.

**TLDR**: When using a function in Bluesky e.g. `yield from x` x must do at least one `yield from `